include/defines.php

```diff
@@ -0,0 +1,16 @@
+<?php
```

Comment:
I moved the defines to here. They were originally in several places, a couple of them were duplicated. The duplication was causing errors in the unit tests.